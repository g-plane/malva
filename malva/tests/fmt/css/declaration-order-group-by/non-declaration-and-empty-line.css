/*cfg declarationOrder = "alphabetical"
      declarationOrderGroupBy = "nonDeclarationAndEmptyLine"*/
div {
  z: 0;
  y: 0;
  x: 0;
  & {}
  f: 0;
  e: 0;

  d: 0;
  c: 0;
  /* b */
  b: 0;
  a: 0;

  & {}
}

div {
  font-family-primary: 'Arial';

  color-primary: #00ff00;
  color-primary-rgb: 0, 255, 0;

  color-secondary: #0000ff;
  color-secondary-rgb: 0, 0, 255;
}
