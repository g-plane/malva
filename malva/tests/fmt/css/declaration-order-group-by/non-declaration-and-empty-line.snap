---
source: malva/tests/fmt.rs
---
/*cfg declarationOrder = "alphabetical"
      declarationOrderGroupBy = "nonDeclarationAndEmptyLine"*/
div {
  x: 0;
  y: 0;
  z: 0;
  & {}
  e: 0;
  f: 0;

  a: 0;
  /* b */
  b: 0;
  c: 0;
  d: 0;

  & {}
}

div {
  font-family-primary: "Arial";

  color-primary: #00ff00;
  color-primary-rgb: 0, 255, 0;

  color-secondary: #0000ff;
  color-secondary-rgb: 0, 0, 255;
}
